src/styles/base/_typography.scss

```
@@ -15,12 +15,32 @@ body {
     font-size: $text-base;
 }
 
-$hero-heading-sm: 40px;
-$hero-heading-md: 56px;
-$hero-heading-lg: 69px;
+.h-primary {
```

```suggestion
.heading--primary {
```
or we can create heading class and nest --primary and --secondary taking out common styles